console/src/lineplot/link.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+import { create, State } from "@/lineplot/slice";
+import { Link } from "@/link";
+
+export const linkHandler: Link.Handler = ({
```

Comment:
I would honestly make the link handler be an async function that way we can use the `await` syntax and not `.then`